crates/ra_assists/src/split_import.rs

```diff
@@ -15,9 +15,7 @@ pub(crate) fn split_import(ctx: AssistCtx<impl HirDatabase>) -> Option<Assist> {
     let top_path = generate(Some(path), |it| it.parent_path()).last()?;
 
     let use_tree = top_path.syntax().ancestors().find_map(ast::UseTree::cast);
-    if use_tree.is_none() {
-        return None;
-    }
+    use_tree?;
```

Comment:
The explicit version looks more readable to me. We can condense this to a single cryptic expression

```
top_path.syntax().ancestors().find_map(ast::UseTree::cast)?
```

but this seems unobvious to me.